### PR TITLE
Accordion enhancement

### PR DESCRIPTION
**Pull request by DevYuns**
## Description

rebuilding an Accordion component. 

## Related Issues

https://github.com/dooboolab/dooboo-ui/issues/199

## Tests

not yet.

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [ ] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [ ] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/Accordion/index.tsx [hunk 1]
```diff
@@ -1,196 +1,98 @@
...
-  isContentVisible: boolean;
-  currentAnimation: AnimationType;
-}
-
-enum AnimationType {
-  showContent = 'showContent',
-  hideContent = 'hideContent',
-  null = 'null',
-}
+type EVProps = {
```
Comment (hyochan) [inline main/Accordion/index.tsx]: What does `EV` stands for?

File: main/Accordion/index.tsx [hunk 2]
```diff
@@ -1,196 +1,98 @@
...
-    )) return;
-    const contentHeight = evt.nativeEvent.layout.height;
-    setContentHeight(contentHeight);
-    setMounted({ ...isMounted, content: true });
-  };
+    if (props.isAnimated) {
+      Animated.timing(animValue, {
+        toValue: targetValue,
+        duration: props.duration || 300,
+      }).start();
```
Comment (hyochan) [inline main/Accordion/index.tsx:73]: Better to return it here and remove the `else` statement.

File: stories/dooboo-ui/Accordion.stories.tsx [hunk 3]
```diff
@@ -29,28 +46,34 @@ const Default = (): React.ReactElement => {
...
-      title: 'Thrid thing',
-      body: `What about very long text? What about very long text?
-        What about very long text? What about very long text?
-        What about very long text? What about very long text?
-        What about very long text? What about very long text?
-        What about very long text? What about very long text?
-        What about very long text? What about very long text?`,
+      title: 'Title 3',
+      body:
+      'Hi. I love this component. What do you think?',
```
Comment (hyochan) [inline stories/dooboo-ui/Accordion.stories.tsx]: What if changing some texts for example? Like a very long text?

File: main/Accordion/index.tsx (hunk 1 above)
Comment (devethan) [reply main/Accordion/index.tsx]: It means short for `ElementValueProps`.
It should be changed.

File: stories/dooboo-ui/Accordion.stories.tsx (hunk 3 above)
Comment (DevYuns) [reply stories/dooboo-ui/Accordion.stories.tsx]: > What if changing some texts for example? Like a very long text?

Okay. I will add some more examples.

File: main/Accordion/index.tsx (hunk 1 above)
Comment (hyochan) [reply main/Accordion/index.tsx]: I think the better naming could be `LayoutProps`.

**Review by hyochan — APPROVED** [review]
LGTM!

**Pull request merged**